artemis-core-client/src/main/java/org/apache/activemq/artemis/core/protocol/core/impl/wireformat/SessionCreateConsumerMessage.java

```diff
@@ -95,6 +109,7 @@ public void encodeRest(final ActiveMQBuffer buffer) {
       buffer.writeNullableSimpleString(filterString);
       buffer.writeBoolean(browseOnly);
       buffer.writeBoolean(requiresResponse);
+      buffer.writeInt(priority);
```

Comment:
I will write it as a byte, if we don't plan to support more then 127 priorities, consumer-side: but as I've said is a negligible save of space on the wire

Reply:
we plan to support int.